README.md

```diff
@@ -1,43 +1,58 @@
 # Instrument Addon Custom Device
 
-The **Instrument Addon Custom Device** allows users to define instrument commands generically. It supports: NI-VISA(GPIB, RS232, RS485, RS422), TCP, UDP, AK TCP, and AK RS232 as base protocols. The communication structure should be command-response. The instrument doesn't initiate any of the communications. Communication rates should be 10s of Hz, though the add-on supports decimation so the NI VeriStand PCL can still be fast.
+The **Instrument Addon Custom Device** allows users to generically define instrument commands.
 
-### Quick Start Documentation
+The add-on supports the following base protocols:
+- NI-VISA (GPIB, RS232, RS485, and RS422)
+- TCP
+- UDP
+- AK TCP
+- AK RS232
 
-- Built: %Public%\Documents\National Instruments\NI VeriStand (Year)\Custom Devices\Instrument Addon\Windows\Quick Start Documentation
-- Source: Source\Quick Start Documentation
+You should set the communication structure to *command-response*. The instrument does not initiate communications. Communication rates should fall between 10 and 19 Hertz. The add-on supports decimation for a fast VeriStand Primary Control Loop.
 
-## LabVIEW Version
+## Quick Start Documentation
 
-LabVIEW 2018
+The following table displays locations where you can access quick start documentation for the Instrument Addon Custom Device.
+
+| Type      | Location |
+| ----------- | ----------- |
+| Built | `%Public%\Documents\National Instruments\NI VeriStand (Year)\Custom Devices\Instrument Addon\Windows\Quick Start Documentation` |
+| Source | `Source\Quick Start Documentation` |
 
 ## Dependencies
 
+### LabVIEW Version
+
+The Instrument Addon Custom Device was built with LabVIEW 2018.
+
 ### Running the add-on
 
-- NI VeriStand >= 2018
-- NI VISA >= 18.0
-- NI Serial >= 18.0
+To run the Instrument Addon Custom Device, you will need to install the following:
+- VeriStand 2018 or later
+- NI VISA 18.0 or later
+- NI Serial or 18.0 later
 
 ### Source
 
+To source for the Instrument Addon Custom Device includes:
```

Comment:
The below is a list of dependencies that you need to install if you want to check the source code, modify or build the custom device locally.

Maybe we can make this more clear in the sentencing, like we did in the previous section (Running the add-on).